src/decorators.ts

```diff
@@ -48,4 +49,24 @@ export function Exclude(options?: ExcludeOptions) {
         const metadata = new ExcludeMetadata(object instanceof Function ? object : object.constructor, propertyName, options || {});
         defaultMetadataStorage.addExcludeMetadata(metadata);
     };
+}
+
+/**
+ * Return the object with the exposed properties only.
+ */
+export function JsonView(params?: {}, method?: string): Function {
```

Comment:
`method?: string` should be a string literal type, e.g. `"classToPlain"|"plainToClass"|...`. This will make it type-safe, predictable and prevent you doing `const classTransformer: any` below.